styles.css

```diff
@@ -0,0 +1,91 @@
+
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: auto;
+  font-family: 'Poppins', sans-serif !important;
+}
+
+.container {
+  height: 100vh;
+  width: 100%;
+  background: url("/img/header-background-mobile.png");
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.container .logo {
+  color: #6070ff;
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: 700;
+  margin-left: 15%;
+  margin-top: 10px;
+}
+
+.menu-bar i {
+  position: absolute;
+  left: 80%;
+  top: 1%;
+}
+
+.container main {
+  width: 100%;
+  height: 100vh;
+  background: url("/img/bg.jpeg");
+  background-size: 100% 100%;
+  background-repeat: no-repeats;
+  display: flex;
+  align-items: center;
+}
+
+main .infoText {
+  padding: 0 20%;
+}
+
+.infoText h1 {
+  color: #091e42;
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 1px;
+  text-transform: capitalize;
+}
+
+.infoText p {
+  color: #344563;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 28px;
+  font-style: normal;
+  font-family: 'Poppins', sans-serif !important;
+}
+
+.infoText h2 {
+  color: #7f8cff;
+  font-size: 16px;
+  font-weight: 500;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.social-media {
```

Comment:
- [ ] Please make sure that you display the social media icons using the Flex Box. You should have `display: flex;` here. It allows you to display your list items better and easily.